epigraphhub/data/_config.py

```diff
@@ -27,3 +29,6 @@
 
 # SINAN data config:
 SINAN_LOG_PATH = "/tmp/sinan_fetch.log"
+_sinan_data = Path().home() / "pysus"
+_sinan_data.mkdir(exist_ok=True)
+SINAN_DATA_PATH = str(_sinan_data)
```

Comment:
I suggest that this variable could be rename to `PYSUS_DATA_PATH` instead, because if in the future we download other datasets via PySUS, we can reuse this cache directory